libs/design/modal/src/modal/modal.component.ts

```diff
@@ -33,13 +33,32 @@ import { DaffModalService } from '../service/modal.service';
   styleUrls: ['./modal.component.scss'],
   animations: [daffFadeAnimations.fade],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
 })
 export class DaffModalComponent implements AfterContentInit, AfterViewInit {
   /**
    * HostBinding to set the default modal class on the host element.
    */
   @HostBinding('class.daff-modal') modalClass = true;
 
+  /**
+   * HostBinding to set the role to dialog.
+   */
+  @HostBinding('attr.role') role = 'dialog';
+
+  /**
+   * HostBinding to set aria-modal to true.
+   */
+  @HostBinding('attr.aria-modal') ariaModal = true;
+
+  private _ariaLabelledBy = null;
+
+  @HostBinding('attr.aria-labelledby') get ariaLabelledBy() {
```

Comment:
Docs. Point out how this is taken by default from the title.

Reply:
added